src/array.js

```diff
@@ -42,8 +54,16 @@ function doubleArray(arr) {
  *    [] => []
  */
 function getArrayOfPositives(arr) {
-	throw new Error("Not implemented");
+  let posarr=[];
+	for (let i=0;i < arr.length ; i++ ) {
+    if (arr[i]>0) {
+      posarr.push(arr[i]);
+    }
+  }
+  return posarr ;
 }
+console.log(getArrayOfPositives([ 0, 1, 2, 3, 4, 5 ]));
```

Comment:
Delete all these `console.log`s from all the test files as well. They can cause the test suite to break undetermistically, and clog up the testing window screen.